.bazelrc

```diff
@@ -75,12 +75,22 @@ build:linux --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
 build:clang-base --//bazel/cc_toolchains:compiler=clang
 build:clang-base --//bazel/cc_toolchains:libc_version=glibc_host
 
+# We build our own chroot for the sysroot tests, which doesn't work well under bazel's sandbox.
+build:sysroot_common --//bazel/test_runners:test_runner=sysroot_chroot
+test:sysroot_common --strategy TestRunner=standalone
+test:sysroot_common --run_under="bazel/test_runners/sysroot_chroot/test_runner.sh"
```

Comment:
The other terminology we have used is <...>-base. We should probably keep this consistent.